service-api/src/main/java/greencity/service/ubs/UBSClientService.java

```diff
@@ -366,4 +366,13 @@ public interface UBSClientService {
      * @author Max Nazaruk
      */
     List<LocationSummaryDto> getLocationSummary();
+
+    /**
+     * Method makes address actual (default) for current {@link User}.
```

Comment:
This method make address with `addresseId` actual for the user with `uuid` not do r current user